### PR TITLE
Updates to CLI commands in the Validation Guide to be compatible with 3.2

### PR DESCRIPTION
**Pull request by etsauer**
@JaredBurck @sabre1041 @oybed

### CODE REVIEW

**Comment by etsauer** [comment]
@JaredBurck bump.

File: playbooks/installation/install_validation.adoc [hunk 1]
```diff
@@ -109,16 +111,20 @@ Follow the steps below
...
 # oc get pods | grep build
 [ copy the pod id ]
-# oc logs <pod id>
+# oc logs -f <pod id>
 [ Look for message saying an image was successfully pushed ]
-# oc get deployments
-[ make sure one has succeeded ]
+# oc get deploymentConfig
+[ copy the deploymentConfig name ]
+# oc deploy <deploymentConfig>
```
Comment (JaredBurck) [inline playbooks/installation/install_validation.adoc]: @etsauer executing `oc deploy <deploymentConfig>` displays the latest deployment for the named DC.
```
$ oc deploy nexus
nexus deployment #2 deployed 10 hours ago - 1 pod
```
add --latest flag to start a new deployment, or --retry flag retry the latest failed deployment, or the --cancel flag to cancel an in progress deployment 

All else looks good to me.

Comment (etsauer) [inline playbooks/installation/install_validation.adoc]: @JaredBurck that's exactly what I'm trying to do here.. view the latest deployment to ensure it succeeded.

Comment (JaredBurck) [inline playbooks/installation/install_validation.adoc]: @etsauer ah gotcha, I misunderstood the intent.  All looks good. OK to merge?

**Comment by etsauer** [comment]
@JaredBurck please merge.

**Pull request merged**